ts/src/test/static/request/bitstamp.json

```diff
@@ -5,27 +5,19 @@
     "methods": {
         "fetchBalance": [
             {
-                "description": "Fetch spot Balance",
+                "description": "Fetch main balance",
                 "method": "fetchBalance",
-                "url": "https://www.bitstamp.net/api/v2/balance/",
-                "input": [
-                    {
-                        "type": "spot"
-                    }
-                ],
-                "output": "type=spot"
+                "url": "https://www.bitstamp.net/api/v2/account_balances/",
+                "input": [],
+                "output": "foo=bar"
             },
             {
-                "description": "Fetch swap Balance",
+                "description": "Fetch subAccount balance",
                 "method": "fetchBalance",
-                "url": "https://www.bitstamp.net/api/v2/balance/",
-                "input": [
-                    {
-                        "type": "swap"
-                    }
-                ],
-                "output": "type=swap"
-            }
+                "url": "https://www.bitstamp.net/api/v2/account_balances/",
+                "input": [],
+                "output": "foo=bar"
```

Comment:
this foo bar is a bit strange

Reply:
nvm 
![image](https://github.com/ccxt/ccxt/assets/43336371/e8098b2e-6d64-4ece-bce2-768a73e6db3c)